lectures/cobweb.md

```diff
@@ -277,7 +277,7 @@ def plot45(model, pmin, pmax, p0, num_arrows=5):
 
     ax.plot(pgrid, g(model, pgrid), 'b-',
             lw=2, alpha=0.6, label='g')
-    ax.plot(pgrid, pgrid, lw=1, alpha=0.7, label='45')
+    ax.plot(pgrid, pgrid, lw=1, alpha=0.7, label='$45\degree$')
```

Comment:
Should this be "45 degrees"?

Reply:
@jstac this notation renders as
<img width="682" alt="Screenshot 2024-05-09 at 10 00 18 AM" src="https://github.com/QuantEcon/lecture-python-intro/assets/8263752/23743f4f-d89d-4677-bd11-3189a7e19b04">
which is a short hand version. I think the text `degrees` may be more conventional but less compact. This comes down to a style choice.

Reply:
@jstac I have no preference on the style here. I would be OK with `degrees` in text over the symbol.

Reply:
Thanks @mmcky .  I like `\degree`, it's a clear improvement.  Please feel free to override me or make small choices like this without my input.